### ISSUE TITLE
geiger counter debug assert

### ISSUE THREAD

**Issue by deltanedas** [space-wizards/space-station-14#16635]
## Description
geiger counter sound plays during prediction, add check before playing sound

**Reproduction**
1. make singulose
2. geiger counter on
3. bonk

```
[ERRO] audio: Tried to create audio source outside of prediction!
[ERRO] net.predict: Failed to apply comp state: entity=23122, comp=Content.Shared.Radiation.Components.GeigerComponent
Unhandled exception. Robust.Shared.Utility.DebugAssertException: Exception of type 'Robust.Shared.Utility.DebugAssertException' was thrown.
   at Robust.Shared.Utility.DebugTools.Assert(Boolean condition) in RobustToolbox/Robust.Shared/Utility/DebugTools.cs:line 36
   at Robust.Client.GameObjects.AudioSystem.TryCreateAudioSource(AudioStream stream, IClydeAudioSource& source) in RobustToolbox/Robust.Client/GameObjects/EntitySystems/AudioSystem.cs:line 306
   at Robust.Client.GameObjects.AudioSystem.Play(AudioStream stream, EntityUid entity, Nullable`1 fallbackCoordinates, Nullable`1 audioParams) in RobustToolbox/Robust.Client/GameObjects/EntitySystems/AudioSystem.cs:line 382
   at Robust.Client.GameObjects.AudioSystem.Play(String filename, Filter playerFilter, EntityUid entity, Boolean recordReplay, Nullable`1 audioParams) in RobustToolbox/Robust.Client/GameObjects/EntitySystems/AudioSystem.cs:line 529
   at Content.Client.Radiation.Systems.GeigerSystem.UpdateGeigerSound(EntityUid uid, Boolean isEnabled, Nullable`1 user, GeigerDangerLevel dangerLevel, Boolean force, GeigerComponent component) in Content.Client/Radiation/Systems/GeigerSystem.cs:line 93
   at Content.Client.Radiation.Systems.GeigerSystem.OnHandleState(EntityUid uid, GeigerComponent component, ComponentHandleState& args) in Content.Client/Radiation/Systems/GeigerSystem.cs:line 30
   at Robust.Shared.GameObjects.EntityEventBus.<>c__DisplayClass49_0`2.<SubscribeLocalEvent>g__EventHandler|0(EntityUid uid, IComponent comp, TEvent& args) in RobustToolbox/Robust.Shared/GameObjects/EntityEventBus.Directed.cs:line 290
   at Robust.Shared.GameObjects.EntityEventBus.<>c__DisplayClass58_0`1.<EntSubscribe>b__0(EntityUid uid, IComponent comp, Unit& ev) in RobustToolbox/Robust.Shared/GameObjects/EntityEventBus.Directed.cs:line 393
   at Robust.Shared.GameObjects.EntityEventBus.DispatchComponent[TEvent](EntityUid euid, IComponent component, CompIdx baseType, Unit& args, Boolean dispatchByReference) in RobustToolbox/Robust.Shared/GameObjects/EntityEventBus.Directed.cs:line 595
   at Robust.Shared.GameObjects.EntityEventBus.Robust.Shared.GameObjects.IDirectedEventBus.RaiseComponentEvent[TEvent](IComponent component, TEvent& args) in RobustToolbox/Robust.Shared/GameObjects/EntityEventBus.Directed.cs:line 162
   at Robust.Client.GameStates.ClientGameStateManager.HandleEntityState(EntityUid uid, IEventBus bus, EntityState curState, EntityState nextState, GameTick lastApplied, GameTick toTick, Boolean enteringPvs) in RobustToolbox/Robust.Client/GameStates/ClientGameStateManager.cs:line 1124
   at Robust.Client.GameStates.ClientGameStateManager.ApplyEntityStates(GameState curState, GameState nextState) in RobustToolbox/Robust.Client/GameStates/ClientGameStateManager.cs:line 735
   at Robust.Client.GameStates.ClientGameStateManager.ApplyGameState(GameState curState, GameState nextState) in RobustToolbox/Robust.Client/GameStates/ClientGameStateManager.cs:line 620
   at Robust.Client.GameStates.ClientGameStateManager.ApplyGameState() in RobustToolbox/Robust.Client/GameStates/ClientGameStateManager.cs:line 305
   at Robust.Client.GameController.Tick(FrameEventArgs frameEventArgs) in RobustToolbox/Robust.Client/GameController/GameController.cs:line 538
   at Robust.Client.GameController.<StartupContinue>b__51_0(Object sender, FrameEventArgs args) in RobustToolbox/Robust.Client/GameController/GameController.cs:line 203
   at Robust.Shared.Timing.GameLoop.Run() in RobustToolbox/Robust.Shared/Timing/GameLoop.cs:line 218
   at Robust.Client.GameController.ContinueStartupAndLoop(DisplayMode mode) in RobustToolbox/Robust.Client/GameController/GameController.Standalone.cs:line 133
   at Robust.Client.GameController.GameThreadMain(DisplayMode mode) in RobustToolbox/Robust.Client/GameController/GameController.Standalone.cs:line 118
   at Robust.Client.GameController.<>c__DisplayClass92_0.<Run>b__0() in RobustToolbox/Robust.Client/GameController/GameController.Standalone.cs:line 85
```

**Comment by themias** [comment]
related to #15180

**Comment by lzk228** [comment]
fixed in #18282
